test/functional/unit_bypass_validation.test.js

```diff
@@ -29,17 +29,18 @@ describe('bypass document validation', function () {
             ok: 1,
             cursor: {
               firstBatch: [{}],
-              id: 23,
+              id: 0,
               ns: 'test.test'
             }
           });
         } catch (e) {
+          console.log('error', e);
```

Comment:
do we need to keep the console logs here and in L91?

Reply:
Removed.

Reply:
Missed the one in L91, but I took the liberty of deleting it myself since it's the only outstanding thing here to merge

Reply:
I also filed a follow up [NODE-3469](https://jira.mongodb.org/browse/NODE-3469) for the other clean up